src/app/routes.js

```diff
@@ -7,6 +7,7 @@ const About = React.lazy(() => import("about/about"))
 const Talent = React.lazy(() => import("talent/talent"))
 const Process = React.lazy(() => import("process/process"))
 const FAQ = React.lazy(() => import("faq/faq"))
+const blog = React.lazy(() => import("blog/blog"))
```

Comment:
blog should be capitalized...in react, we use PascalCase to name files and components.